target/linux/mediatek/modules.mk

```diff
@@ -41,7 +41,7 @@ $(eval $(call KernelPackage,sdhci-mtk))
 
 define KernelPackage/crypto-hw-mtk
   TITLE:= MediaTek's Crypto Engine module
-  DEPENDS:=@TARGET_mediatek
+  DEPENDS:=@TARGET_mediatek @arm
```

Comment:
I would prefer `DEPENDS:=@TARGET_mediatek_mt7623 @TARGET_mediatek_mt7629` here.
The @arm looks strange to me if you make it depend on the sub targets it is clear that this depends on the subtargets.

Reply:
I agree.

I have just now noticed that the driver is compiled into the mt7623 kernel: 
https://github.com/openwrt/openwrt/blob/7a256d97d9ded84d1bfd531e775099774e7b6b06/target/linux/mediatek/mt7623/config-5.10#L114

So the package ends up being built for mt7629 only. So `@TARGET_mediatek_mt7629` should suffice.